### PR TITLE
Ruby up

### PR DESCRIPTION
**Pull request by jefk**

### CODE REVIEW

**Comment by jefk** [comment]
some other commits made it in

File: views/layout.haml [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
     %meta{ content: 'width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0', name:'viewport' }
     %meta{ name: 'apple-mobile-web-app-capable', content: 'yes' }
+    %meta{ name: 'apple-mobile-web-app-status-bar-style', content: 'black' }
```
Comment (jshou) [inline views/layout.haml]: what does the status bar do?

**Comment by jshou** [comment]
:ship: 

do you have heroku permissions? i don't remember

**Comment by jefk** [comment]
yep it's already on heroku before the :ship: =)

File: views/layout.haml (hunk 1 above)
Comment (jefk) [inline views/layout.haml:8]: https://developer.apple.com/library/iad/documentation/AppleApplications/Reference/SafariHTMLRef/Articles/MetaTags.html#//apple_ref/doc/uid/TP40008193-SW4

**Pull request merged**